recipe/meta.yaml

```
@@ -27,17 +27,24 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - maturin 0.14.17
+    - maturin 1.3.1
   run:
     - python
 
 test:
+  source_files:
+    - tests
   imports:
     - cramjam
   commands:
     - pip check
+    - pytest -v -n auto
```

```suggestion
    - pytest -v -n auto tests
```
I like it when we are explicit about which folder/files pytest should be run against. If we don't do that and it's wrongly configured (via config files or whatever), then it could pick more tests then really initially desired.